string  experiment
int32   seq
string  sensors
bool    arm
bool    angular
---
bool    received